addons/web/static/src/model/relational_model/relational_model.js

```diff
@@ -302,20 +295,30 @@ export class RelationalModel extends Model {
         if (config.isRoot) {
             this.hooks.onWillLoadRoot(config);
         }
+
         if (config.isMonoRecord) {
             const evalContext = getBasicEvalContext(config);
             if (!config.resId) {
                 return this._loadNewRecord(config, { evalContext });
             }
-
-            const records = await this._loadRecords(
-                {
-                    ...config,
-                    resIds: [config.resId],
-                },
-                evalContext
-            );
-            return records[0];
+            try {
+                const records = await this._loadRecords(
+                    {
+                        ...config,
+                        resIds: [config.resId],
+                    },
+                    evalContext
+                );
+                return records[0];
+            } catch (err) {
+                const route = { ...this.router.current };
+                delete route.hash.id;
+                delete route.hash.view_type;
```

Comment:
what if we are in a dialog?

Also, you could maybe use the action service directly instead of playing with the url: `action.restore()` would display the previous view/action in the breadcrumbs

Reply:
There is, the multi record view is there in the breadcrumb